### ISSUE TITLE
Defense bot mostly goes after teammates and keeps assaulting them forever

### ISSUE THREAD

**Issue by Udrakan** [Regalis11/Barotrauma#10550]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Everytime I made a defense bot it started shooting at allies and it would never stop unless I took the ammo out. 

### Reproduction steps

Use the defense bot in multiplayer

### Bug prevalence

Happens every time I play

### Version

Faction/endgame test branch

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Not specific to the faction branch

**Comment by itchyOwl** [comment]
I've been testing and debugging this for a while now (both mp and sp), but haven't found anything, except that the required angle to shoot was too high, which resulted in the bot starting to shoot before the turret was pointing towards the target. Fixed that in Regalis11/Barotrauma-development@161e08d. Based on this and another report, there might be more to it.

Specific steps to repro the issue would be appreciated. What happened before the defensebot starts to shoot at friendlies? Is the shooting systematic (= is it attacking or just missing the real target)? If it happens also in single player, please take a shot with the "debugai" info enabled and post it here.

**Comment by Kaktar-1** [comment]
Awhile ago I found that I could get it to shoot me if I played Engineer and grabbed the bot. I figure grabbing it inflicted a stun which caused it to get aggressive. It happened twice in a row with different engineers but didn't work for any of the other jobs I had around. It fired in short controlled shots every once in awhile and didn't unload the entire magazine -- as though it were playing with me at my dispense. I honestly thought it was a feature.
